docs/doc.main

```diff
@@ -18,14 +18,12 @@
   ],
 
   "Rings" => [
-     "AbstractAlgebra/rings.md",
+     "AbstractAlgebra/ring.md",
```

Comment:
It dies with error
> ERROR: LoadError: 'AbstractAlgebra/ring.md' is not an existing page!

Perhaps this requires a newer AA version and `Project.toml` should be changed to require that version?

Reply:
Yes.